server.js

```diff
@@ -8,7 +8,7 @@ if (!process.env.DEMO_DEPLOY) {
 }
 
 const server = require('./app');
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 5000;
```

Comment:
splitting this off to be the server portion that serves up the statically built assets rather than compiling on the fly. so now there is the client `npm start` and the server `node server.js`, they can't run on the same port. we do the same thing in tooling